lib/chat/getChatContext.tsx

```diff
@@ -13,64 +13,72 @@ import getRecentScore from "./getRecentScore";
 import getScoresInPast24 from "./getScoresInPast24";
 import getFans from "./getFans";
 
-const getChatContext = async () => {
+const getChatContext = async (isHabitQuestion: boolean) => {
```

Comment:
I like how you're thinking of creating different `instructions` based on which `function` the user is intending to `invoke`.

Would it be a more scalable approach to use `function calling`?
- actual: `habits` is only triggered on `suggestion pill click`
- expected: AI is used to determine if the `habitQuestion` is being invoked.

https://openai.com/index/function-calling-and-other-api-updates/

![Screenshot 2024-10-10 at 4 36 05 PM](https://github.com/user-attachments/assets/875c8986-832f-4a42-abdb-c80d7fcbd820)